CIP-token-metadata-translations/README.md

```diff
@@ -0,0 +1,139 @@
+---
+CIP: ?
+Title: Extend token metadata for translations
+Category: Metadata
+Status: Proposed
+Authors:
+    - Vito Melchionna <info@granadapool.com>
+    - Aaron Schmid <aaron@fortech.group>
+    - Carolina Isler @LaPetiteADA <lapetiteada@granadapool.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/488
+Created: 2023-03-19
+License: CC-BY-4.0
+---
+
+## Abstract
+This proposal defines an additional property to the metadata standard for tokens (NFTs and FTs) to support text localization.
+
+## Motivation: why is this CIP necessary?
+Current token metadata only supports a single hardcoded language (mostly English), which limits the accessibility to a certain culture. To get closer to mass adoption, we need to bring down language barriers by extending the current standard to support translations. This is especially relevant for games, metaverse solutions, and RealFi use cases of NFTs.
+
+## Specification
+This proposal follows the same specifications as [CIP-0025](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025).
+    
+The name of a culture consists of its [[ISO-639]](https://www.iso.org/standard/4767.html) language code with small letters and its [[ISO-3166]](https://www.iso.org/standard/63545.html) country/region code with capital letter separated by a dash "-". For instance, this proposal was written in "en-US": English with the US culture.
+
+This convention is compatible with most operative systems (Linux and Windows) and widely used translation software.
+    
+### Proposed structure

```

Comment:
As this contains on-chain data, I believe it must also have a CDDL specification (see [here](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md?plain=1#L83C541-L83C620)).

Reply:
Done!